sys/aarch64/timer.c

```diff
@@ -80,7 +80,7 @@ static int arm_timer_attach(device_t *dev) {
   };
 
   state->irq_res =
-    bus_alloc_irq(dev, 0, BCM2836_INT_CNTPNSIRQ_CPUN(0), RF_ACTIVE);
+    bus_alloc_irq(dev, 0, BCM2836_INT_CNTPNSIRQ_CPUN(0), RF_ACTIVE | RF_SHAREABLE);
```

Comment:
Why we need that flag here?

Reply:
Should this interrupt be exclusive to the timer?

Reply:
I thought about "what is a difference now". Previously it just worked so what is a implication of that flag?

Reply:
It doesn't really make a difference for now. It allows future sharing of the `BCM2836 INT_CNTP SIRQ CPU(0)` interrupt. I added it in order to remove any confusions associated with any future attempt to share this interrupt. Do you think that we should delay this change to some future PR?

Reply:
As long as it works I don't care but from my point of view the best what we can do now is discuss that offline with @cahirwpz and then make a decision.

Reply:
After offline discussion – let it be.